src/window.js

```diff
@@ -108,3 +109,5 @@ function save({ file, value, readonly }) {
     }
   }
 }
+
+
```

Comment:
Please keep line breaks as they were (including the deleted one above) and in general it's a good idea to avoid unrelated/unintended changes